src/index.js

```diff
@@ -340,7 +340,10 @@ function transformGlimmer(ast, { env }) {
         return
       }
 
-      node.value = sortClasses(node.value, { env })
+      node.value = sortClasses(node.value, {
+        env,
+        ignoreLast: parent.type === 'SubExpression' && !/[^\S\r\n]/.test(node.value.at(-1)),
```

Comment:
Thinking out loud:

```hbs
<div class='{{echo-first-argument-and-discard-second "border-l-4 border-" @borderColor}}'></div>
<div class='{{echo-first-argument-and-discard-second "border-l-4 border- " @borderColor}}'></div>
```

Would produce different outputs although from the perspective of Tailwind the values are the same.

Since we can't infer what will call `(foo @arg1 @arg2)` return, as it's effectively a black box from the perspective of the linter, I'd say we can't say for sure what will be returned to the calling context if we call a _helper_ function. Therefore those should not be sorted.

But maybe for trivial cases like ternary `class="{{if @isTrue 'px-2 py-4' 'mx-4 my-8'}}"` it would be still ok?